sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.4.0-beta.1 (2024-07-12)
```

Comment:
could we include https://github.com/Azure/azure-sdk-for-net/pull/44754 so as to not delay this important gap for another release? @vishweshbankwar mentioned that it was decided to use the same categoryname property name in exporter as well.

Reply:
Let me follow up with Vishwesh. I'm fine waiting till Monday to do the release, but I don't want to wait longer than that.